### PR TITLE
Updated the share to twitter so it includes both user's twitter handles.

### PR DESCRIPTION
**Pull request by AdamMc331**
Tested by starting the share process for a talk with one speaker and a talk with multiple. 

Would have written unit tests, but last time we tried mocking the firebase it didn't work out, so I couldn't reliably write any.

### CODE REVIEW

**Comment by n8ebel** [comment]
@fetching  do you have time to test this?

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt [hunk 1]
```diff
@@ -252,13 +252,11 @@ class AgendaDetailFragment : Fragment() {
...
         tv_agenda_detail_shareText.setOnClickListener {
-            val twitter = viewModel.getSpeaker(scheduleDetail.listRow.primarySpeakerName)?.socialProfiles?.get("twitter")
-
-            var twitterVal =  if(twitter.isNullOrEmpty()) scheduleDetail.listRow.primarySpeakerName else "@$twitter"
+            val twitterVal = viewModel.getTwitterHandleForAllSpeakers(scheduleDetail)
 
             val tweetUrl = "https://twitter.com/intent/tweet?text=I really enjoyed this %23droidconbos talk \"${scheduleDetail.listRow.talkTitle}\"  by $twitterVal"
             val uri = Uri.parse(tweetUrl)
-            var shareIntent = Intent(Intent.ACTION_VIEW, uri)
+            val shareIntent = Intent(Intent.ACTION_VIEW, uri)
```
Comment (n8ebel) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt]: 👍

**Comment by fetching** [comment]
Yes I will be able to test it tomorrow evening.

**Comment by AdamMc331** [comment]
Cool. Will wait to merge. Are we in the code freeze stage already? I understand if we don't wanna include this.

**Comment by n8ebel** [comment]
@AdamMc331  Let's consider this the last thing to go in.  If this can be tested by @fetching  tonight, then let's merge it and I'll upload a new apk to the store.

Since nothing major came up in the last round of testing, we should be good to roll that out

**Comment by AdamMc331** [comment]
@n8ebel @fetching I've made small tweaks to this after further testing. I removed an extra space that was in the string, and I also made it so that we hide this share to twitter for talks that don't have a speaker. 

Did that because we don't want people to share how much they loved check in, or lunch. 😆 

I went into a detail fragment for talks with no speakers, talks with one speaker, and talks with two speakers. I feel confident in this. I'm going to leave the PR open for now, in case any night owls or early birds want to also give it a look. 

but in the AM, if you're ready to push up a new build feel free to merge this in. :)

**Pull request merged**